runtime/criusupport/criusupport.hpp

```diff
@@ -28,12 +28,6 @@
 
 extern "C" {
 
-jboolean JNICALL
-Java_openj9_internal_criu_InternalCRIUSupport_isCheckpointAllowedImpl(JNIEnv *env, jclass unused);
-
-jboolean JNICALL
-Java_openj9_internal_criu_InternalCRIUSupport_isCRIUSupportEnabledImpl(JNIEnv *env, jclass unused);
-
```

Comment:
These declarations should have been moved to `jclprots.h` rather than just deleted.

Reply:
Is there a functionality requirement such DDR parsing for these declarations in `jclprots.h` cause no other references until native binding phase?

Reply:
Declarations such as those at the very least provide a double-check of the signature (by the compiler). It has nothing to do with DDR.